source/transfer_list.rst

```diff
@@ -902,5 +902,42 @@ This manifest contains the SPMC attribute node consumed by the SPMD
      - hdr_size
      - Holds a SPMC manifest image in DT format.
 
+
+**Executable entry point information entry layout (XFERLIST_EXEC_EP_INFO)**
+
+This entry type holds an array of `entry_point_info_t` structures.
+`entry_point_info_t` is a data structure containing information used by a
```

Comment:
I think we need a little more info here. This range is used for all TF projects and is not exclusive to TF-A, so you should say here that this is specifically a TF-A project structure. Also, it would be good to say this contains info like the PC, SPSR, GP regs (like you do in the commit message).

Ideally (but not essential for this PR), it would be nice to refer to a TF-A document that explains how BL31 uses this entry_point_info_t array to execute the bootflow.

Reply:
I've added a reference to [1](https://trustedfirmware-a.readthedocs.io/en/latest/design/firmware-design.html#data-structures-used-in-the-bl31-cold-boot-interface) which explains at a high level how BL31 uses `entry_point_info_t` in the cold boot flow, and added details about its contents i.e. register representation.